extension/src/popup/components/PasswordRequirements/styles.scss

```diff
@@ -2,15 +2,15 @@
   padding: 1.25rem 0 2.5rem;
 
   &__list {
-    color: rgba(255, 255, 255, 0.6);
-    list-style-type: disc;
+    color: var(--color-gray-60) !important;
+    list-style-type: disc !important;
     list-style-position: inside;
     font-size: 1rem;
     padding: 0;
     margin: 0;
 
-    li::before {
-      content: none;
+    ::before {
+      content: none !important;
```

Comment:
we need to manually add this for all lists to clear out SDS default styling. We can figure out how to address this later